weave/Scripts/Player.cs

```diff
@@ -73,21 +81,16 @@ public override void _Ready()
 
     public override void _Process(double delta)
     {
-        if (_hasReachedSize)
-        {
-            return;
-        }
+        if (!_hasReachedSize)
+            Scale = Scale.Lerp(_desiredScale, (float)delta * 4f);
 
-        Scale = Scale.Lerp(_desiredScale, (float)delta * 4f);
-
-        // Have not reached size yet
-        if (Mathf.Abs(Scale.DistanceTo(_desiredScale)) > 0.01f)
+        if (Mathf.Abs(Scale.DistanceTo(_desiredScale)) < 0.01f)
```

Comment:
Put this in the same if statement above? So we don't have to check if it has reached its desired scale when we know that it has